app.py

```diff
@@ -50,14 +49,7 @@ def get_data():
 # ------------------------------
 st.write('''# Project Tornado''')
 
-try:
-    df, fatal_loss = get_data()
-except ClientError as e:
-    st.error("We're sorry, but our bandwidth cap has been reached for the day.  Please come again tomorrow!\
-        If this problem persists, please contact one of us via our GitHub: https://github.com/Vigneshwarr3/Tornado_project")
```

Comment:
We'll need this in case we go over data limits.  It also is part of testing requirement